deploy/vagrant/provision/roles/spark/files/spark_env.sh

```diff
@@ -4,5 +4,5 @@ cp /vagrant/files/workers /spark/conf/slaves
 
 cat > /spark/conf/spark-env.sh << EOF
 export SPARK_MASTER_IP=$(tail -n1 /spark/conf/slaves)
-export SPARK_LOCAL_HOSTNAME=$(hostname)
+export SPARK_LOCAL_HOSTNAME=\`hostname\`
```

Comment:
Otherwise the value will be evaluated to AlluxioMaster and then rsynced with the incorrect value.